### DISCUSSION TITLE
Can't proceed Copilot payment in Belarus.

### DISCUSSION THREAD

**Discussion by LazavikouArtsiom** [community/community#31210] [Copilot Conversations]
`Cannot proceed with payment
We are still investigating the issue with your GitHub account, and we will update you as soon as possible. Thank you for your patience during this process. We are sorry for any inconvenience.`

So i've tried to pay from one card and something went wrong, so i want to try other card from other bank, but i can't because i see this problem :(

**Comment by jfullstackdev** [comment]
is that a reply from your opened ticket ? 

if not, you may want to try that 

https://support.github.com/contact/account

**Comment by grigoryevandrey** [comment]
I have the same problem...
I guess we need to create a new account

**Reply by dutsik** [reply]
I got it after review. Not sure if Belarus is really eligidle, or they just don't want to bother. SHould I create an appeal request? or supprot ticket?
![image](https://user-images.githubusercontent.com/584244/206923741-8ea275cf-4b12-4ed8-a54f-f8a9e8550042.png)

**Comment by RomanPreis** [comment]
I've run into this issue while trying to sponsor an open-source contributor
